packages/checkbox/src/useCheckbox.ts

```diff
@@ -1,23 +1,181 @@
-import { ref, computed, inject } from 'vue'
+import {
+  ref,
+  computed,
+  inject,
+  getCurrentInstance,
+  onMounted,
+  watch,
+} from 'vue'
+import { toTypeString } from '@vue/shared'
+import { UPDATE_MODEL_EVENT } from '@element-plus/utils/constants'
+import { ICheckboxGroupInstance, ICheckboxProps, IArgs } from './checkbox'
 
-export const useCheckbox = () => {
+export const useCheckboxGroup = () => {
   //todo: ELEMENT
   const ELEMENT = null
   const elForm = inject('elForm', {})
   const elFormItem = inject('elFormItem', {}) as any
-  const _checkboxGroup = inject('CheckboxGroup', {}) as any
-  const focus = ref(false)
-  const isGroup = computed(() => _checkboxGroup && _checkboxGroup.name === 'ElCheckboxGroup')
-  const _elFormItemSize = computed(() => {
+  const checkboxGroup = inject<ICheckboxGroupInstance>('CheckboxGroup', {})
+  const isGroup = computed(() => checkboxGroup && checkboxGroup.name === 'ElCheckboxGroup')
```

Comment:
I wonder when `checkboxGroup` will be falsy.

Reply:
It's a question -_-